content/en/docs/instrumentation/java/automatic/spring-boot.md

```diff
@@ -2,28 +2,40 @@
 title: Spring Boot
 linkTitle: Spring Boot
 weight: 30
-description: Spring instrumentation for OpenTelemetry Java
+description: Spring Boot instrumentation for OpenTelemetry Java
 cSpell:ignore: autoconfigure datasource logback springboot springframework
 ---
 
-You can use the [OpenTelemetry Java agent](..) with byte code instrumentation to
-automatically instrument a [Spring Boot](https://spring.io/projects/spring-boot)
-application; or you can also use the OpenTelemetry [Spring Boot starter] to
-instrument your application.
+The [OpenTelemetry Java agent](..) with byte code instrumentation can cover most
+of your needs when instrumenting
+[Spring Boot](https://spring.io/projects/spring-boot) applications.
+
+You can't use the OpenTelemetry Java agent with Spring Boot Native image
+applications, but you still can instrument your code using OpenTelemetry [Spring
+Boot starter].
 
 [Spring Boot starter]:
   https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using.build-systems.starters
 
-The OpenTelemetry starter is compatible with Spring Boot 2.0 and 3.0, and Spring
-native.
+The
+[opentelemetry-java-examples/spring-native](https://github.com/open-telemetry/opentelemetry-java-examples/tree/main/spring-native)
+repository contains an example of a Spring Boot Native image application
+instrumented using the OpenTelemetry Spring Boot starter.
+
+Another situation in which the OpenTelemetry starter can help you is if the
+startup overhead of the OpenTelemetry Java agent exceeds your requirements.
+
+If your application already uses another Java monitoring agent, the OpenTelemetry Java agent
+might not work. In this case, use the OpenTelemetry starter.
```

Comment:
what do you think of making these 3 bullets? I think it might give this section a bit of helpful structure for readers